CMakeLists.txt

```diff
@@ -114,13 +133,30 @@ if(ENABLE_LTO)
     set(CMAKE_INTERPROCEDURAL_OPTIMIZATION TRUE)
 endif()
 
-if(ENABLE_OPENMP)
-    find_package( OpenMP REQUIRED )
-    include(FindOpenMP)
```

Comment:
Disappeared? What does it do?

Reply:
This moved down, and got disabled for macOS, since that now compiles OpenMP itself (as a static library)